wharehauora-wifi-gateway/wharehauora-wifi-gateway.ino

```diff
@@ -6,11 +6,32 @@
 
 const char my_ssid[] = "";
 const char my_pass[] = "";
-const char my_server[] = "m11.cloudmqtt.com";
+
+#ifdef STAGING
+  // Staging MQTT server
+  const char my_server[] = "m11.cloudmqtt.com";
+  #define MY_PORT 16259 //not-ssl
+  //#define MY_PORT 26259 /// SSL
+#else
+  // Production MQTT server
+  const char my_server[] = "m12.cloudmqtt.com";
+  #define MY_PORT 14876 //not-ssl
+#endif
 
 char mqtt_username[32];
 char mqtt_password[32];
 
+/* Variable for setting up the publish topic  */
+#define ZERO 48
```

Comment:
this can be written this way as well `const int zero = 48;` right? is there any specific reason to use #define?

Reply:
kinda. that would allocate some memory and put the number 48 there.

but when we use `#define`, the compiler instead swaps any `ZERO` used in the code to be `48` - so it becomes like a hard coded number, so the compiled code is smaller. (We only have 1MByte to use on the device).

Reply:
OOOHH! It shouldn't say variable :-) it's not a variable. I'll fix that.

Reply:
oh! thanks for the explanation ⭐️

Reply:
/* Start and end position in username string to read when setting up the publish topic  */